cmake/cmake-3.5.2-msys.patch

```diff
@@ -26,27 +26,27 @@ diff -Naur cmake-2.8.11.2/bootstrap cmake-2.8.11.2-msys/bootstrap
  # Add CoreFoundation framework on Darwin
  if ${cmake_system_darwin}; then
    cmake_ld_flags="${LDFLAGS} -framework CoreFoundation"
-@@ -701,6 +712,7 @@
+@@ -763,6 +774,7 @@
  case "${cmake_system}" in
    *AIX*)   cmake_toolchains='XL GNU' ;;
    *CYGWIN*) cmake_toolchains='GNU' ;;
 +  *MSYS*) cmake_toolchains='GNU' ;;
    *Darwin*) cmake_toolchains='GNU Clang' ;;
    *Linux*) cmake_toolchains='GNU Clang XL PGI PathScale' ;;
    *MINGW*) cmake_toolchains='GNU' ;;
-diff -Naur cmake-2.8.11.2/CMakeCPack.cmake cmake-2.8.11.2-msys/CMakeCPack.cmake
---- cmake-2.8.11.2/CMakeCPack.cmake	2013-07-02 17:41:39.000000000 +0400
-+++ cmake-2.8.11.2-msys/CMakeCPack.cmake	2014-04-06 22:14:23.853800000 +0400
-@@ -56,6 +56,8 @@
+diff -aur cmake-3.5.2/CMakeCPack.cmake.orig cmake-3.5.2/CMakeCPack.cmake
+--- cmake-3.5.2/CMakeCPack.cmake.orig	2016-04-15 09:41:19.000000000 -0400
++++ cmake-3.5.2/CMakeCPack.cmake	2016-05-22 17:29:34.072190400 -0400
+@@ -54,6 +54,8 @@
      # cygwin is good for the system name
-     if("${CMAKE_SYSTEM_NAME}" STREQUAL "CYGWIN")
+     if("x${CMAKE_SYSTEM_NAME}" STREQUAL "xCYGWIN")
```

Comment:
Wouldn't this `if("x${CMAKE_SYSTEM_NAME}" STREQUAL "xCYGWIN")` break Cmake for CYGWIN?